pages/main-experience.tsx

```diff
@@ -0,0 +1,54 @@
+import Button from "components/button/button"
+import CustomDivComponent from "components/CustomDivComponent"
+
+function MainExperience() {
+  return ( <>
+    <CustomDivComponent 
+    className = "sampleDiv"
+    color = "green"
+    header = "first div"
+    height = "500px"
+    width = "100%"
+    >
+      blah
+      <Button onClick={() => console.log("you clicked the cat!")}
+        backgroundURL = "/cat.png"
+        height = "200px"
+        width = "200px"
+        > 
+      </Button>
```

Comment:
let's change the props to be indented and also move the onclick prop to it's own line. Since the button doesn't take in children, we can used this notation
```
   <Button 
        onClick={() => console.log("you clicked the cat!")}
        backgroundURL = "/cat.png"
        height = "200px"
        width = "200px"
    /> 
```

Reply:
let's also make sure to do this on the other occurrences of this in this file